index.html

```diff
@@ -91,28 +91,28 @@ <h2>A Note on Feature Detection</h2>
   </p>
   <aside class="example" title="Checking existence of ComputePressureObserver interface">
     <p>
-      This simple example illustrates how to check whether the User Agent exposes the ComputePressureObserver interface
+      This simple example illustrates how to check whether the User Agent exposes the
+      ComputePressureObserver interface
     </p>
     <pre class="js">
       if (typeof ComputePressureObserver == "function") {
         // use ComputePressureObserver interface
       }
 
-      if ("ComputePressureObserver" in window) {
+      if ("ComputePressureObserver" in globalThis) {
         // use ComputePressureObserver interface
       }
 
-      if (window.ComputePressureObserver) {
+      if (globalThis.ComputePressureObserver) {
         // use ComputePressureObserver interface
       }
-
-      // etc.
     </pre>
   </aside>
   <p>
     All of these tell you something about the presence and possible characteristics of an API.
-    They do not tell you anything, however, about whether that API is actually connected to a real [=platform collector=], whether the [=platform collector=] is collecting real telemetry readings,
-    or whether the user is going to allow you to access it.
+    They do not tell you anything, however, about whether that API is actually connected to a
+    real [=platform collector=], whether the [=platform collector=] is collecting real telemetry
+    readings, or whether the user is going to allow you to access it.
```

Comment:
I didn't see in the spec where user could not allow it.

Reply:
True, we haven't looked into whether we want to guard this behind a permission and where that should be done - mind filing an issue?

Reply:
https://github.com/WICG/compute-pressure/issues/44